modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/service/impl/RemoteAppEntryLocalServiceImpl.java

```diff
@@ -34,6 +34,7 @@
 import com.liferay.portal.kernel.search.SearchContext;
 import com.liferay.portal.kernel.search.SearchException;
 import com.liferay.portal.kernel.search.Sort;
+import com.liferay.portal.kernel.security.auth.GuestOrUserUtil;
```

Comment:
Not this code, but the commit message. It has a typo :sweat_smile: :

> when de app 

should be:

> when the app

Reply:
😂 I will fix it